processor/src/decoder/tests.rs

```diff
@@ -1371,44 +1371,44 @@ fn dyn_block() {
     // --- check hasher state columns -------------------------------------------------------------
 
     // in the first row, the hasher state is set to hashes of both child nodes
-    let join_hash: Word = join_node.digest().into();
-    let dyn_hash: Word = dyn_node.digest().into();
+    let join_hash: Word = join_node.digest();
+    let dyn_hash: Word = dyn_node.digest();
     assert_eq!(join_hash, get_hasher_state1(&trace, 0));
     assert_eq!(dyn_hash, get_hasher_state2(&trace, 0));
 
     // in the second row, the hasher set is set to hashes of both child nodes of the inner JOIN
-    let mul_bb_node_hash: Word = mstorew_node.digest().into();
-    let save_bb_node_hash: Word = push_node.digest().into();
+    let mul_bb_node_hash: Word = mstorew_node.digest();
+    let save_bb_node_hash: Word = push_node.digest();
     assert_eq!(mul_bb_node_hash, get_hasher_state1(&trace, 1));
     assert_eq!(save_bb_node_hash, get_hasher_state2(&trace, 1));
 
     // at the end of the first SPAN, the hasher state is set to the hash of the first child
     assert_eq!(mul_bb_node_hash, get_hasher_state1(&trace, 4));
-    assert_eq!([ZERO, ZERO, ZERO, ZERO], get_hasher_state2(&trace, 4));
+    assert_eq!(Into::<Word>::into([ZERO, ZERO, ZERO, ZERO]), get_hasher_state2(&trace, 4));
```

Comment:
Same comment as above here and in other similar places.

Reply:
Fixed